setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+setup(
+    name='vkjokes',
+    version='0.1',
+    py_modules=['vkjokes'],
+    install_requires=[
+        'click',
+    ],
+    entry_points={
+        'console_scripts': [
+            'vkjokes = vkjokes:cli',
+            ],
+    },
+)
```

Comment:
Were you attempting to package your cli project as a pip dependency? Is that why this and the `vkjokes.egg-info/` files are included?

Reply:
Nice, having read the docs this looks good 👍 . I actually didnt know about this and it solves a problem I have always faced when using click.